### PR TITLE
Miscellaneous updates to wiki pages

### PR DESCRIPTION
**Pull request by seanlip**
This PR makes a bunch of miscellaneous updates to the wiki pages:

- Drop references to CircleCI, which we no longer use
- Put the team onboarding guides in a single place in the sidebar
- Change "Interpreting CircleCI Results" to "Interpreting GitHub Actions Results" in the sidebar
- Rewrite the debugging docs page -- move the debugging doc template to the top and integrate the existing instructions within that template
- Update "Revert and regression policy" page to explain that 'revert PRs' can mostly skip the review process, and that the reverter does not need to get permission from the PR author before reverting
- Add more tutorials for CSS, JS and Angular. Drop tutorials for AngularJS.
- Replace admin@oppia.org with dev-support@oppia.org where appropriate.
- For GSoC, explain that "beginner to open source" doesn't mean "beginner to coding"
- Onboarding:
  - Add to issue assignment instructions: state when you expect to open a PR
  - In PR process: explain how to check that a PR is linked to the corresponding issue
  - In PR process: explain how to find unresolved comments in the GitHub conversation tab
  - In PR process: add https://github.com/oppia/oppia/wiki/How-to-access-Oppia-webpages to help devs understand how to test things locally
- Add tip from @srijanreddy98 for how to run frontend unit tests faster

### CODE REVIEW

File: Revert-and-Regression-Policy.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ Once the dev workflow leads decide whether to revert, fix forward, or both, you
 
 Any Oppia developer with permission to open revert PRs on GitHub may decide, following the above policy, whether to revert the PR, fix forward, pursue both in parallel, or make one of the two allowed exceptions. If you are such a developer, you should document your reasoning and communicate on the tracking issue so the dev workflow leads can understand what happened.
 
+Please note that:
+* You don't need to get permission from the author of the original PR before making your revert PR. Just notify them so that they are aware.
+* It is usually fine to merge revert PRs without review, as long as the relevant CI checks pass. Contact one of the dev workflow leads (or **@seanlip**) to help you with this.
```
Comment (U8NWXD) [inline Revert-and-Regression-Policy.md]: Only seanlip can override review requirements, so the dev workflow leads can't help with that part

File: Instructions-for-making-PR-passes.md [hunk 2]
```diff
@@ -54,36 +54,11 @@ On Github:
...
 
 ## Rota
 
 Dev-workflow team members are expected to take a [full pass](#full-pass) on a weekly basis and create a report which the team lead can share with Core-maintainers in the Core-maintainer's meeting.
 
 **Current Dev-workflow team members taking pass:** @sajalasati & @DubeySandep
 
-**Weekly report doc:** https://docs.google.com/document/d/1lisXJYYXbmLGLGWoJx0E0XxAMndVG0J98KMp8N-hOho
\ No newline at end of file
+**Weekly report doc:** https://docs.google.com/document/d/1lisXJYYXbmLGLGWoJx0E0XxAMndVG0J98KMp8N-hOho
```
Comment (U8NWXD) [inline Instructions-for-making-PR-passes.md]: This can be removed since we don't do it anymore

File: If-CI-checks-fail-on-your-PR.md [hunk 3]
```diff
@@ -36,9 +36,9 @@ Traceback (most recent call last):
     "__main__", fname, loader, pkg_name)
   File "/usr/local/lib/python2.7/runpy.py", line 72, in _run_code
     exec code in run_globals
-  File "/home/circleci/oppia/scripts/install_third_party_libs.py", line 311, in <module>
+  File "scripts/install_third_party_libs.py", line 311, in <module>
     main()
-  File "/home/circleci/oppia/scripts/install_third_party_libs.py", line 236, in main
+  File "scripts/install_third_party_libs.py", line 236, in main
```
Comment (U8NWXD) [inline If-CI-checks-fail-on-your-PR.md]: Maybe add `...` to the paths to make clear that users might see a different prefix

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: Revert-and-Regression-Policy.md (hunk 1 above)
Comment (seanlip) [reply Revert-and-Regression-Policy.md]: Done.

File: Instructions-for-making-PR-passes.md (hunk 2 above)
Comment (seanlip) [reply Instructions-for-making-PR-passes.md]: Done.

File: If-CI-checks-fail-on-your-PR.md (hunk 3 above)
Comment (seanlip) [reply If-CI-checks-fail-on-your-PR.md]: Done.

**Comment by seanlip** [comment]
@U8NWXD Addressed the comments, PTAL. Thanks!

File: Instructions-for-making-PR-passes.md [hunk 4]
```diff
@@ -54,36 +54,9 @@ On Github:
...
-   If you don't see the "Cancel" button, you can't cancel this build like the following:
-
-   ![CI Action Disabled](images/circleCIWorkflowActionDisabled.png)
-
 
 ## Rota
 
 Dev-workflow team members are expected to take a [full pass](#full-pass) on a weekly basis and create a report which the team lead can share with Core-maintainers in the Core-maintainer's meeting.
 
 **Current Dev-workflow team members taking pass:** @sajalasati & @DubeySandep
```
Comment (U8NWXD) [inline Instructions-for-making-PR-passes.md]: I was referring to this whole "Rota" section (you can see from how the snippet above my comment in the PR thread had extra diff lines that I had selected this whole section, not just the last line) in https://github.com/oppia/oppia-web-developer-docs/pull/187#discussion_r1292894416

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: Instructions-for-making-PR-passes.md (hunk 4 above)
Comment (seanlip) [reply Instructions-for-making-PR-passes.md]: Ah, sorry, I didn't realize that! Thanks, done.

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**